src/test/java/com/google/api/codegen/testdata/discogapic/java/java_simplecompute.v1.json.baseline

```diff
@@ -3112,11 +3112,9 @@ public final class DeleteAddressHttpRequest implements ApiMessage {
     }
 
     public final AddressName getAddressAsAddressName() {
-      java.lang.String str = getAddress();
-      if (str.isEmpty()) {
-        return null;
-      }
-      return AddressName.parse(str);
+      return AddressName.create(
+          getAddress(),        getProject(),        getRegion()
```

Comment:
Could you change the template to add a newline on the end of each of these? Formatting doesn't have to be perfect in our generated code since we run the formatter, but I'd like it to be a little better than this.

Reply:
:)